src/detect-pangram.js

```diff
@@ -15,7 +15,16 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const letters = 'abcdefghijklnmopqrstuvwxwz';
+  if (phrase.length < 1) {
```

Comment:
думаю в этой проверке нет необходимости
```suggestion
  if (!phrase.length) {
```